components/NotificationPopup/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+
+import { useGetNotifications } from "@/hooks/useGetNotifications";
+
+import styles from "./NotificationPopup.module.scss";
+import BaseNotification from "../Notification/BaseNotification";
+import EmptyNotification from "../Notification/EmptyNotification";
+
+const cn = classNames.bind(styles);
+
+export default function NotificationPopup() {
+  const [isClicked, setIsClicked] = useState(false);
+
+  const { data: allNotifications } = useGetNotifications();
+
+  const handleBellClick = () => {
+    setIsClicked(!isClicked);
+  };
+
+  return (
+    <div className={cn("container")}>
+      <figure onClick={handleBellClick} className={cn("notification-bell")}>
+        <figcaption className={cn("notification-count")}>{allNotifications?.count}</figcaption>
+        <Image src="/icons/notification-bell.svg" width={26} height={20} alt="알림 종" priority />
+      </figure>
+      {isClicked && (
+        <div className={cn("notification-popup")}>
+          {allNotifications?.count ? (
```

Comment:
```suggestion
          {allNotifications?.count ? (
```
P3: `count`는 읽지 않은 알림의 개수여서 렌더링과는 무관할 것 같아요!
예를 들어서 알림을 다 읽어서 `count`가 0인 경우에는 알림이 없다고 나오는 이슈가 있을 것 같네요